src/GalleryTools/Commands/BackfillTfmMetadataCommand.cs

```diff
@@ -28,10 +31,34 @@ public static void Configure(CommandLineApplication config)
         protected override List<string> ReadMetadata(IList<string> files, NuspecReader nuspecReader)
         {
             var supportedTFMs = new List<string>();
-            var supportedFrameworks = _packageService.GetSupportedFrameworks(nuspecReader, files);
+
+            // We wrap this int a try catch because fetching supported frameworks on existing packages can sometimes throw due to format errors
+            // (usually portable TFM formatting). In this case we'll clear the TFMs.
+            var supportedFrameworks = Enumerable.Empty<NuGetFramework>();
+            try
+            {
+                supportedFrameworks = _packageService.GetSupportedFrameworks(nuspecReader, files);
+            }
+            catch (Exception e)
```

Comment:
We can catch a more specific exception here, right? I think it's a specific one that is caused by the bad portables.

Reply:
```suggestion
            catch
```
if we end up using a generic catch-all like the other places

Reply:
It's a little generic--it's an `ArgumentException` which is why I opted for catching all. But I'm happy to make it explicit as exceptions not of this type will fall through and be logged, which could be useful.